tests/configure/test_config.py

```diff
@@ -78,3 +81,38 @@ def test_command(): # noqa
                                              TEST_PROFILE_2])
     assert '--profile can only be provided once. The profiles [{}, {}] were provided.'.format(
         TEST_PROFILE_1, TEST_PROFILE_2) in result.output
+
+
+TEST_HOST = 'https://test.cloud.databricks.com'
+TEST_TOKEN = 'testtoken'
+
+
+def test_command_headers():
+    @click.group()
+    @config.profile_option
+    def test_group():
+        pass
+
+    @click.group()
+    @config.profile_option
+    def test_group_2():
+        pass
+
+    @click.command()
+    @config.profile_option
+    @config.provide_api_client
+    def test_command(api_client): # noqa
+        click.echo(json.dumps(api_client.default_headers))
+
+    with mock.patch("databricks_cli.configure.provider.DatabricksConfig") as ConfigMock:
+        with mock.patch("uuid.uuid1") as uuid_mock:
+            ConfigMock.return_value = DatabricksConfig.from_token(TEST_HOST, TEST_TOKEN)
```

Comment:
Why do you need to mock this class here while other tests don't? Could you add some comments here?

Reply:
When I mock it, any instantiation of the DatabricksConfig will be valid in order to properly return the API client.